libraries/postgres/src/Database.js

```diff
@@ -423,7 +423,7 @@ class Database {
     await this._withClient('admin', async client => {
       // check the DB collation by its behavior, rather than by name, as names seem to vary.
       for (const pair of ['aA', 'ab', 'Ab', 'aB', 'AB', '0a', '0A']) {
-        const res = await client.query(`select 1 where $1 >= $2`, [pair[0], pair[1]]);
+        const res = await client.query(`select 1 where $1 > $2`, [pair[0], pair[1]]);
```

Comment:
I'm not sure I understand. Wouldn't changing '>=' to '>' only make a difference if we are comparing the same character (e.g 'a' > 'a'). None of the pairs have the same character as far as I can see.

In other words, running `select 1 where 'a' > 'A'` and `select 1 where 'a' >= 'A'` both returns `1`. How would that solve ricky's issue?

Reply:
> In other words, running `select 1 where 'a' > 'A'` and `select 1 where 'a' >= 'A'` both returns `1`. How would that solve @ricky26's issue?

For a _correctly_ configured database: `select 1 where 'a' > 'A'` should not return any results, whereas `select 1 where 'a' >= 'A'` should return `1` since `'a'` and `'A'` are considered equal when it comes to ordering in a case-insensitive database.

For an _incorrectly_ configured database: `select 1 where 'a' > 'A'` may return `1` (for example if it has case-sensitive ordering), and `select 1 where 'a' >= 'A'` can also return `1` for the same reason.

Since we only want to return results for incorrectly configured databases, we therefore use the first form (`>`) which does _not_ return `1` for a correctly configured database. 

I put some context in #3598 that goes into a bit more detail.